src/pkg/lint/validate_schema.go

```diff
@@ -0,0 +1,58 @@
+package lint
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+const (
+	zarfInvalidPrefix = "zarf.yaml is not valid:"
+	zarfWarningPrefix = "zarf schema warning:"
+	zarfTemplateVar   = "###ZARF_PKG_TMPL_"
```

Comment:
It might be better to make this a variable that can be shared since this magic string is also used with `promptAndSetTemplate`

Reply:
Exported it for now, maybe a better place to put magic strings like this going forward? english.go didn't feel right

Reply:
Didn't end up exporting as I moved to packager package anyway.